docker-cmd-mac.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Example of docker usage for RLLIB + SUMO Utlis
+#
+# Author: Lara CODECA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+
+set -e
+set -u
+
+IMAGE_NAME="tf-gpu-sumo-$(date +%Y-%m-%d)"
+IMAGE_FOLDER="docker-image-mac"
+GPU=true
+GPU_OPT="--gpus all"
+BUILD=false
+CACHE=false
+RUN=false
+SCREEN=false
+EXEC=false
+CONTAINER=""
+DEVEL_DIR=""
+LEARN_DIR=""
+USER_ID=""
+GROUP_ID=""
+COMMAND=""
+EXP=""
+DETACH=false
+
```

Comment:
I had to add a new parameter, due to the warning issued by the deep RL infrastructure.
See https://github.com/lcodeca/rllibsumodocker/commit/fd5ea7aeb4071e7d96774e2a6d98030eac896fa5

Reply:
synchronised now 7d3d34938ac9834901c248367e653a367b3e77d5